### DISCUSSION TITLE
算法名称加粗的问题

### DISCUSSION THREAD

**Discussion by pkeke** [note286/xduts#100] [Q&A] [answered]
请问一下如何加粗下面的红色部分：
![fe1024fcf3f94bef4013f51b6466e34](https://user-images.githubusercontent.com/44956389/227720693-4ac3eddc-5094-4d33-927f-29ee36d6636f.png)

**Comment by note286** [comment, marked as answer]
参考 https://github.com/note286/xduts/discussions/85 中的回复。

**Reply by pkeke** [reply]
谢谢！
